[2025-01-28T12:16:14.786+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T12:16:14.801+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T12:12:42.277857+00:00 [queued]>
[2025-01-28T12:16:14.807+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T12:12:42.277857+00:00 [queued]>
[2025-01-28T12:16:14.808+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 3
[2025-01-28T12:16:14.819+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): ingest_data> on 2025-01-28 12:12:42.277857+00:00
[2025-01-28T12:16:14.825+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_data_ingestion', 'ingest_data', 'manual__2025-01-28T12:12:42.277857+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/dag_data_ingestion.py', '--cfg-path', '/tmp/tmp32s9z1wn']
[2025-01-28T12:16:14.827+0000] {standard_task_runner.py:105} INFO - Job 95: Subtask ingest_data
[2025-01-28T12:16:14.828+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1363) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-28T12:16:14.829+0000] {standard_task_runner.py:72} INFO - Started process 1364 to run task
[2025-01-28T12:16:14.863+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T12:12:42.277857+00:00 [running]> on host 0711bf72f9ce
[2025-01-28T12:16:14.937+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T12:12:42.277857+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-28T12:12:42.277857+00:00'
[2025-01-28T12:16:14.937+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T12:16:14.948+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-28T12:16:14.949+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "2025-01"']
[2025-01-28T12:16:14.959+0000] {subprocess.py:99} INFO - Output:
[2025-01-28T12:16:14.961+0000] {subprocess.py:106} INFO - 2025-01
[2025-01-28T12:16:14.961+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-28T12:16:14.984+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T12:16:14.984+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_data_ingestion, task_id=ingest_data, run_id=manual__2025-01-28T12:12:42.277857+00:00, execution_date=20250128T121242, start_date=20250128T121614, end_date=20250128T121614
[2025-01-28T12:16:15.002+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T12:16:15.016+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T12:16:15.017+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
